README.md

```diff
@@ -2,30 +2,29 @@
 
 
 ## Overview
+OpenFin Notifications provides developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
 
-OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
-
-Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the system tray notification icon.
+Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the Notifications icon in the system tray.
 
 This project consist of 3 parts:
-1. The Notifications Service, displaying and managing notifications and owning the Notification Center UI
-2. The Notifications Client, exposing API's for applications to create and manage notifications
+1. The Notifications Provider, displaying and managing notifications, and owning the Notification Center
+2. The Notifications Client, exposing APIs for applications to create and manage notifications
 3. The Notifications Demo App, demonstrating the different features of OpenFin Notifications
 
 ### Dependencies
 - OpenFin version for applications using Notifications >= 9.61.38.41
-- OpenFin version used in the Notifications Service = 10.66.39.43
+- OpenFin version used in the Notifications service = 10.66.39.43
```

Comment:
I thought we settled on "Notifications Service" as a proper noun at some point. If not, we should decide and I argue for both capitalised.

Reply:
Gone back to 'Notifications Service', but changed the opening sentence slightly - feel like if it's a proper noun, we should be defining it

Reply:
Cool.